content/cpp/concepts/math-functions/terms/isfinite/isfinite.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: 'isfinite()'
+Description: 'Returns whether the argument is a finite value.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+Returns whether the argument is a finite value.
+
+## Syntax
+
+```cpp
+isfinite(x)
+```
+
+## Example
+
+Use `isfinite()` to check whether `5` is finite:
+
+```cpp
+#include <iostream>
+#include <cmath>
+
+int main() {
+  int result;
+
+  result = std::isfinite(5.0);
+
+  if(result == 0){
+    std::cout << "Not finite" << "\n";
+  }
+  else{
+    std::cout << "Is finite" << "\n";
+  }
+  // Output: Is finite
+}
+```
+
+## Codebyte Example
+
```

Comment:
```suggestion

The following example is runnable and features a case where running the `isfinite()` functions returns `false` and the value is not finite:

```

Reply:
I would actually prefer a mix of cases that return both `true` and `false`. Something like this:

```
std::isfinite(4.0);
std::isfinite(1.0/0.0);
std::isfinite(10);
std::isfinite(NAN); // Or std::isfinite(Inf);
```